array-manipulation-2/src/take-right.js

```diff
@@ -0,0 +1,11 @@
+/* exported takeRight */
+function takeRight(array, count) {
+  var newArr = [];
+  if (count >= array.length) {
+    return [];
```

Comment:
If the count is larger than the length, you don't want to return an empty array, you want to return the original array in full.